src/test/helpers/defaults.tsx

```diff
@@ -108,3 +109,22 @@ export const defaultNonAppealProducts: CampaignPolicy[] = [
     ],
   },
 ];
+
+export const defaultCampaignConfig: CampaignConfig = {
+  features: {
+    ...defaultFeatures,
+    campaignName: "Some Campaign Name",
+  },
+  policies: [
+    {
+      category: "toilet-paper",
+      name: "Toilet Paper",
+      order: 1,
+      quantity: {
+        period: 7,
+        limit: 2,
+      },
+    },
+  ],
+  c13n: {},
+};
```

Comment:
we've been manually added the campaigns :| . can take a look at `useTranslate.test.tsx` 😂 we'll update these in tech debtss

Reply:
dont have. 😂 , i think next time instead of `defaultProducts` or `defaultNonAppealProducts`, etc., we should be constructing them instead. e.g.,
```
const getCampaignConfig = (features, policies, c13n) => {...}
```
like that 🤔

Reply:
ok! let's make a note somewhere so we wouldnt forget HAHA